File: JS simple calculator/models/Calculator.js
function Calculator (leftOperand, rightOperand, operator) {
    this.leftOperand = leftOperand;
    this.rightOperand = rightOperand;
    this.operator = operator;

    this.calculatorResult() = function() {
        let result = 0;

        switch (this.operator) {
            case '+':
                result = this.leftOperand + this.rightOperand;
                break;
            case '-':
                result = this.leftOperand - this.rightOperand;
                break;
            case '*':
                result = this.leftOperand * this.rightOperand;
                break;
            case '/':
                result = this.leftOperand / this.rightOperand;
                break;
        }

        return result;
    }
}

File: C# Caclulator/Calculator-CSharp/Models/Calculator.cs
using System;

namespace Calculator_CSharp.Models
{
    public class Calculator
    {
        public decimal LeftOperand { get; set; }
        public decimal RightOperand { get; set; }
        public String Operator { get; set; }
        public decimal Result { get; set; }

        public Calculator ()
        {
            this.Result = 0;
        }
    }
}